### DISCUSSION TITLE
Include printed edition images for lff book lessons

### DISCUSSION THREAD

**Discussion by mtdvlpr** [sircharlo/meeting-media-manager#873] [Ideas] [answered]
Hey @sircharlo, this weeks congregation bible study got me wondering. Might it be useful to include the printed edition images in the media fetch. This week we're studying lesson 32 and it has three tables which consist of a lot of small images in the digital version which are fetched, like the small crown images which no ones is going to use (this is where the hide feature comes in). But in this scenario the printed edition of those tables might be more useful to show. We could also add it as an optional feature to be turned on in the settings. What do you think?

Pros:
- More options to show during the meeting
- Less likely that the brother on stage is asking something that is not fetched

Cons:
- More cluttered media list
- The table is half of the image (could zoom in), so you might still want to clip it yourself and include it as custom media
- More often than not the extra images are not used: unnecessary fetching / time consumption / resource usage

**Comment by sircharlo** [comment, marked as answer]
Discussion continues in #872 😆
